Client/Frontend/Widgets/PhotonActionSheet/AppMenu.swift

```diff
@@ -109,9 +110,11 @@ extension PhotonActionSheetProtocol {
         let noImageMode = PhotonActionSheetItem(title: imageModeTitle, iconString: iconString, isEnabled: noImageEnabled) { action,_ in
             NoImageModeHelper.toggle(isEnabled: action.isEnabled, profile: self.profile, tabManager: self.tabManager)
             if noImageEnabled {
-                TelemetryWrapper.recordEvent(category: .action, method: .tap, object: .blockImagesDisabled)
+                Analytics.shared.browser(.disable, label: .blockImages)
+//                TelemetryWrapper.recordEvent(category: .action, method: .tap, object: .blockImagesDisabled)
             } else {
-                TelemetryWrapper.recordEvent(category: .action, method: .tap, object: .blockImagesEnabled)
+                Analytics.shared.browser(.enable, label: .blockImages)
+//                TelemetryWrapper.recordEvent(category: .action, method: .tap, object: .blockImagesEnabled)
```

Comment:
And this too? Maybe if we want to keep it for future generations we can add a comment :hugs:

Reply:
Same here, commenting might make easier future merges with upstream.